docs/external-template-creation.md

```diff
@@ -37,7 +37,7 @@ Remove extraneous attributes on the base `svg` component and make sure your comp
 
 ```js
 const title = __('Some Descriptor', 'web-stories');
-/* eslint-disable-next-line react/prop-types */
+/* eslint-disable-next-line react/prop-types -- @todo */ 
```

Comment:
Let's disable this rule for docs here:

https://github.com/GoogleForCreators/web-stories-wp/blob/700032264a04185f9ca2b5232e3fe61a8ca34313/.eslintrc#L509-L514